sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/AutoRefreshUserCredential.java

```diff
@@ -3,9 +3,6 @@
 
 package com.azure.android.communication.common;
 
-import com.azure.android.core.credential.AccessToken;
```

Comment:
should we update the build.gradle to remove the core dependency?

Reply:
Removed from gradle